test/test-utils.js

```diff
@@ -33,7 +35,11 @@ export const PLAYER = /** @type {const} */ ({
 });
 
 const names = ['Alice', 'Bob', 'Cathy', 'Donald', 'Emily'];
-const suits = ['Red', 'Yellow', 'Green', 'Blue', 'Purple'];
+const noVar = /** @type {Variant} */ {
```

Comment:
Not sure if this code currently type checks, but in order to perform a type assertion, the expression needs to be wrapped in brackets:
```js
const noVar = /** @type {Variant} */ ({ ... });
```